app/routines/update_user_salesforce_info.rb

```diff
@@ -5,6 +5,13 @@ class UpdateUserSalesforceInfo
     'Career School/For-Profit (2)'
   ]
 
+  ADOPTION_STATUSES = {
+    "Current Adopter" => true,
+    "Future Adopter" => true,
+    "Past Adopter" => false,
+    "Not Adopter" => false
+  }
+
```

Comment:
🤔 How do I know that these exact strings are how they will come always come from Salesforce? If they are correct today, case-sensitive, separated by a space, what if they change in the future? I think there should be an integration test for this, at least; that would appease my concerns.

Reply:
Tests added